Contentstack.Core/Configuration/Config.cs

```diff
@@ -107,6 +108,10 @@ internal string HostURL
             {
                 if (Region == ContentstackRegion.EU)
                     return "cdn.contentstack.com";
+                if (Region == ContentstackRegion.Azure_EU)
```

Comment:
@AnkurContentstack Remove if conditions for Azure_EU and Azure_NA and add it to first if with or conditions:
```
if (Region == ContentstackRegion.EU || Region == ContentstackRegion.Azure_EU || Region == ContentstackRegion.Azure_NA) {
     return "cdn.contentstack.com";
} 
```

Reply:
based on above logic is set for generate final URL this condition is not worked